src/Components/WordTermDisplay.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from 'react';
+import WordTerm from './WordTerm';
+
+// airtable configuration
+const Airtable = require('airtable');
+
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey })
+  .base(airtableConfig.baseKey);
+
+function WordTermDisplay() {
+  const [dictionaryDetails, setDictionaryDetails] = useState([]);
```

Comment:
You can also rename this to something like 'words' or 'definitions' to more clearly notate what's being held by the state! As is, 'dictionary details' sounds like its referring to some metadata about the dictionary itself, like the publisher or language.